.github/workflows/test-k8s-service-buffer.yml

```diff
@@ -86,9 +90,18 @@ jobs:
           sleep 10
           # Dump objects
           /tmp/latest-scripts/k8s/dump.sh /tmp/artifacts/test-k8s-state.new.json
+
       - name: Check deployment of the etcd
         if: always()
-        run: kubectl rollout status "sts/$ETCD_RELEASE_NAME"
+        run: kubectl rollout status "sts/$ETCD_RELEASE_NAME" --timeout=10s
```

Comment:
Toto je aj vo `wait.sh`, takze tu sa to iba overuje, staci kratky timeout.
(ked som hladal chybu, tak ma to zdrzovalo)